test/modules/linear_boltzmann_solvers/transport_steady/transport_2d_triangular_P0.lua

```diff
@@ -0,0 +1,126 @@
+-- 2D LinearBSolver Vacuum with isotropic incident, triangular set
+-- SDM: PWLD
+-- Test: 
+num_procs = 1
+
+--############################################### Check num_procs
```

Comment:
check other test file to see if we still use 
--########################################

Reply:
There are comments and section separations using this notation in the other tests. I removed them in my cases as requested.